index.html

```diff
@@ -1616,14 +1616,12 @@ <h3>Algorithm</h3>
         </li>
         <li>Set the <a>reverse property</a> flag of <var>definition</var>
           to <code>false</code>.</li>
-        <li>If <var>value</var> contains the <a>entry</a> <code>@id</code> and its value
-          does not equal <var>term</var>:
+        <li class="changed">If <var>value</var> contains the <a>entry</a> <code>@id</code> and its value
+          neither equals <var>term</var> nor is <code>null</code>
```

Comment:
Maybe this should end with a ":"?  Otherwise it runs into the inline "note" below.  It'll look odd either way.

Related is that all the other inline "notes" also look odd compared to the full "NOTE" blocks.  Were they supposed to be inline?

Reply:
ReSpec supports both, and sometimes it makes sense to do. Full block notes are pretty heavyweight.

Reply:
I think this still needs something done.  It reads "... nor is null If null, the term ...".  The inline note style is there for the "If null...", but the final rendered result could be improved.